### ISSUE TITLE
Library is generally buggy and non-functional

### ISSUE THREAD

**Issue by JoshWillik** [alongubkin/angular-datepicker#49]
I could not get this port to work at all. I spent almost 2 hours and got nothing.
I love how this library looks, so if you should ever restart maintaining this library, please close this issue so I know I can start using it again.

Thanks :)

**Comment by Nahrae** [comment]
Agreed

**Comment by bramski** [comment]
Thanks for these notes so I don't waste my time.  Cheers.

**Comment by johnculviner** [comment]
Yeah it doesn't even appear to allow you to initialize your '$scope.myDate' to a value. If you do this the library doesn't allow you to select a new date. (SKIP THIS LIB!!)

**Comment by alongubkin** [comment]
https://github.com/alongubkin/angular-datepicker/commit/4737234dbfb3da67e85e53bf1bff378469d83db6
